### ISSUE TITLE
Bloomburrow offspring creatures lack special "offspring token" images

### ISSUE THREAD

**Issue by Svyatoslav28597** [magefree/mage#12904]
To elaborate on the issue: All bloomburrow creatures have special token copies reserved to the paying offspring cost. While offspring seems to be working perfectly in the xmage - all copies just use normal image of the creature. Would it be possible for offspring tokens to use special images, or this would require too much refactoring and fixing problems with things like copying creatues? 
This also may be a bit tricky to add due to things like [[Cottontail Caretaker]] and [[Zinnia, Valley's Voice]] adding offspring to cards which normally don't have it

Offspring Token:
![image](https://github.com/user-attachments/assets/e42035eb-a843-406a-8494-a3b49403c3ef)
Offspring token in game:
![image](https://github.com/user-attachments/assets/164f4a29-9d57-4029-bfa1-8e292bab6af0)

**Comment by JayDi85** [comment]
It’s useless to add cause will support little amount of all possible images. Same as tokens from Embalm ability like [[Sacred Cat]].

**Comment by ssk97** [comment]
> It’s useless to add cause will support little amount of all possible images. Same as tokens from Embalm ability like [[Sacred Cat]].

I strongly disagree that this is useless. Those token images exist, why shouldn't we use them? You fixed this same issue yourself in #3201, no?

**Comment by JayDi85** [comment]
> I strongly disagree that this is useless. Those token images exist, why shouldn't we use them? You fixed this same issue yourself in #3201

Nope, it's old code from old days with direct images control and miss token images from unsupported cards. Embalm tokens doesn't work in current code (it use same logic as offspring -- copied token from a card). You will need to inject images checking and setup code directly to card/ability logic with additional checking of supported and non supported cards. It's weird.

**Comment by ssk97** [comment]
> Nope, it's old code from old days with direct images control and miss token images from unsupported cards. Embalm tokens doesn't work in current code (it use same logic as offspring -- copied token from a card). You will need to inject images checking and setup code directly to card/ability logic with additional checking of supported and non supported cards. It's weird.

It's not currently an easy thing to do, explaining why it's not currently working, but it's something that we would like to have working if we could. Seems to me like having it as an open issue is correct, even if it's not being actively worked on.
